layouts/release/single.html

```diff
@@ -13,17 +13,39 @@
 -->
 
 {{ $checksum_suffix := "md5" }}
+
 <!-- if the version > 0.21 -->
 {{ if .File.BaseFileName | replaceRE "^([0-9]+.[0-9]+).*" "$1" | float | lt 0.21 }}
   {{ $checksum_suffix = "mds" }}
-{{ end }}
-<!-- if the version > 0.20.2 -->
-{{ if .File.BaseFileName | replaceRE "^([0-9]+.[0-9]+).*" "$1" | eq "0.20" }}
+<!-- else if the version > 0.20.2 -->
+{{ else if .File.BaseFileName | replaceRE "^([0-9]+.[0-9]+).*" "$1" | eq "0.20" }}
   {{ if .File.BaseFileName | replaceRE "^[0-9]+.[0-9]+.([0-9]+).*" "$1" | float | lt 2 }}
     {{ $checksum_suffix = "mds" }}
   {{ end }}
 {{ end }}
 
+<!-- if the version >= 3.3 -->
```

Comment:
did we miss branch-3.0 release line?

Reply:
branch-3.0 is EoL, so we can miss it.
https://cwiki.apache.org/confluence/display/HADOOP/EOL+%28End-of-life%29+Release+Branches

Reply:
thanks for the clarification.. I missed this in community.